### PR TITLE
HomeWork

### PR DESCRIPTION
**Pull request by Daniluko**

### CODE REVIEW

File: Lesson10/src/main/resources/lesson10/DataBases.sql [hunk 1]
```diff
@@ -0,0 +1,169 @@
+SELECT * FROM OFFICES;
+COL CITY  FORMAT A16
```
Comment (MateAcademy) [inline Lesson10/src/main/resources/lesson10/DataBases.sql]: what is it?

File: Lesson10/src/main/resources/lesson10/DataBases.sql [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+SELECT * FROM OFFICES;
+SELECT TABLE_NAME FROM USER_TABLES;
+SELECT CITY, TARGET, SALES FROM OFFICES;
+SELECT CITY, TARGET, SALES 
+FROM OFFICES
+WHERE REGION = '�����';
+SELECT CITY, TARGET, SALES
+    FROM OFFICES
+    WHERE REGION = '�����'
```
Comment (MateAcademy) [inline Lesson10/src/main/resources/lesson10/DataBases.sql]: Why no correct record?

File: Lesson10/src/main/resources/lesson10/DataBases.sql [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+ SELECT MFR_ID, PRODUCT_ID, DESCRIPTION, (QTY_ON_HAND * PRICE)
+    FROM PRODUCTS;
+COL STOIM FORMAT A12
+ SELECT MFR_ID, PRODUCT_ID, DESCRIPTION, '$ '||(QTY_ON_HAND * PRICE) AS STOIM
+    FROM PRODUCTS;
+SELECT NAME, QUOTA, (QUOTA +((SALES/100)*3))
+    FROM SALESREPS;    
+SELECT CITY, 'Has sales of', SALES
+    FROM OFFICES ;   
+COL CITY FORMAT A18
```
Comment (MateAcademy) [inline Lesson10/src/main/resources/lesson10/DataBases.sql]: ????

File: Lesson10/src/main/resources/lesson10/DataBases.sql [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+       WHERE COMPANY  = '��������';    
+SELECT COMPANY, CREDIT_LIMIT
+    FROM CUSTOMERS
+    WHERE COMPANY LIKE '%�';
+SELECT COMPANY, CREDIT_LIMIT
+    FROM CUSTOMERS
+    WHERE COMPANY LIKE '%����' ;
+SELECT COMPANY, CREDIT_LIMIT
+    FROM CUSTOMERS
+    WHERE COMPANY LIKE '��_�����';
```
Comment (MateAcademy) [inline Lesson10/src/main/resources/lesson10/DataBases.sql]: Why no correct record too?

File: Lesson10/src/main/resources/lesson10/DataBases.sql (hunk 1 above)
Comment (Daniluko) [reply Lesson10/src/main/resources/lesson10/DataBases.sql]: Format table

**Pull request merged**